en/components/transaction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Transaction Service
+_description: 
+_keywords: Ignite UI for Angular, transaction
+---
+
+
+## Transaction Service
+
+The Transaction Service is an injectable middleware (through Angular's DI) that a component may use to accumulate changes without immediately affecting the underlying data.
+    <img src="https://cdn-images-1.medium.com/max/800/1*O-6DidcFW_XCSqgKRfXf_Q.png"
+        style="display:flex;max-height:400px;margin:auto auto 20px auto;" />
+
+The Transaction Service allows adding a transaction, committing or clearing all changes. As it keeps a detailed log, it can also execute undo and redo operations.
+
+Every time you execute an operation (transaction), it is added to the transaction log and undo stack. All the changes in the transaction log are then accumulated per record. From that point, the service maintains an aggregated state that consists only of add/update/delete operations for unique records.
```

Comment:
we can bold **transaction** and **state** at least in this topic, eventually in all others?